components/service/glean/src/main/java/mozilla/components/service/glean/storages/StorageEngineManager.kt

```diff
@@ -30,6 +32,56 @@ internal class StorageEngineManager(
         }
     }
 
+    /**
+     * Splits a labeled metric back into its name/label parts.
+     *
+     * If not a labeled metric, the second part of the Pair will be null.
+     *
+     * @param key The key for the metric value in the flattened storage engine
+     * @returns A pair (metricName, label).  label is null if key is not
+     * from a labeled metric.
+     */
+    private fun parseLabeledMetric(key: String): Pair<String, String?> {
+        var divider = key.indexOf('/', 1)
```

Comment:
nit: is `var` really required here? We're not ever changing this, so I guess it could be a `val`